### DISCUSSION TITLE
Disqus page identifier is not unique

### DISCUSSION THREAD

**Discussion by patrickdung** [requarks/wiki#4281] [Error / Bug Report]
The page identifier (https://wikijs.domain.name/i/2) is the same for different URLs

```
</div></template></page></div><script>
  var disqus_config = function () {
    this.page.url = 'https://wikijs.domain.name/i/2';
    this.page.identifier = '2';
  };
```

Version is 2.5.201

**Comment by patrickdung** [comment]
Push!

**Reply by ahmadnizar-lap** [reply]
Good evening, please help. I am a Palestinian from Gaza. I need surgery

**Reply by EyalSeagull** [reply]
@patrickdung , I think the issue is solved. Please see #3454

**Reply by patrickdung** [reply]
@EyalSeagull , thanks for the info.

**Comment by patrickdung** [comment]
This is a problem because users may access the URL with /lang/ (e.g /en/) and without /lang/
